File: exercicios/2059/2059.py
n = input().split()

p = int(n[0])
j1 = int(n[1])
j2 = int(n[2])
r = int(n[3])
a = int(n[4])
g1 = 'Jogador 1 ganha!'
g2 = 'Jogador 2 ganha!'

v = 0

if (j1 + j2) % 2 == 0:
    if p == 1:
        v = 1
    else:
        v = 2
else:
    if p == 0:
        v = 1
    else:
        v = 2

if r == 1 and a == 1:
    print(g2)

elif r == 1 and a == 0:
    print(g1)

elif r == 0 and a == 1:
    print(g1)

elif r == 0 and a == 0:
    if v == 1:
        print(g1)
    elif v == 2:
        print(g2)

File: exercicios/JavaScript/2059/2059.js
var input = require('fs').readFileSync('stdin','utf-8');
var lines = input.split('\n');

var v = lines.shift().split(' ');

var p = parseInt(v[0]);
var j1 = parseInt(v[1]);
var j2 = parseInt(v[2]);
var r = parseInt(v[3]);
var a = parseInt(v[4]);

var v = 0

var jg1 = 'Jogador 1 ganha!';
var jg2 = 'Jogador 2 ganha!';

if((j1 + j2)%2 == 0){
    if(p == 1){
        v = 1
    }else{
        v = 2
    }
}else{
    if(p == 0){
        v = 1
    }else{
        v = 2
    }
}

if(r == 1 && a == 1){
    console.log(jg2)
}else if(r == 1 && a == 0){
    console.log(jg1)
}else if(r == 0 && a == 1){
    console.log(jg1)
}else if(r == 0 && a == 0){
    if(v == 1){
        console.log(jg1)
    }else if(v == 2){
        console.log(jg2)
    }
}